### PR TITLE
Theme context to redux

### PR DESCRIPTION
**Pull request by zoef-yt**
[ONLY CHECK ME](https://github.com/zoef-yt/gamers-connect/pull/5/commits/fe88440ae54acf9fe1aff51ab59a781100741d7a)
In this PR, I added 
- Theme context to redux
- Follow unfollow users.
- Fixed auth.

### CODE REVIEW

File: src/Firebase/FirebaseFirestore.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
+import { getFirestore, collection, setDoc, doc, getDoc, getDocs, deleteDoc } from 'firebase/firestore';
+import { app } from './FirebaseConfig';
+import { setFollowing } from '../store/Auth/AuthSlice';
+
+const db = getFirestore(app);
+
+const userCollection = collection(db, 'users');
+// const postCollection = collection(db, 'posts');
```
Comment (iRohitGaur) [inline src/Firebase/FirebaseFirestore.js:8]: suggestion: remove commented code

File: src/Firebase/FirebaseFirestore.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+		await setDoc(doc(userCollection, unfollowingUserId), { followers: followers.size }, { merge: true });
+
+		const currentUserFollowers = await getDocs(currentUserFollowingCollection);
+		const allFollowers = [];
+		currentUserFollowers.forEach((doc) => {
+			allFollowers.push(doc.data());
+		});
+		dispatch(setFollowing(allFollowers));
+	} catch (error) {
+		console.log('un-follow not done', error);
```
Comment (iRohitGaur) [inline src/Firebase/FirebaseFirestore.js:68]: suggestion: remove `console.log` and display error

**Review by iRohitGaur — APPROVED** [review]
Nice work!

Clean code. Good to merge with updates 👍🏻

**Review by itsmebipul00 — APPROVED** [review]
Great work Zoef! Small suggestion for UX is that you could use a scrollbar or an scroll button for friends(header section) in the landing page for accessibility(desktop users). Which could be done later. Amazing layout , Code is good. Good to merge

**Pull request merged**